src/JobDetailModal/JobDetailsModal.scss

```diff
@@ -0,0 +1,46 @@
+@import "../Constants";
+
+.job-detail-modal {
+  width: 60%;
+  position: fixed;
+  top: 20px;
+  left: 50%;
+  transform: translate(-50%);
```

Comment:
width 60% is a bit arbitrary, better to have a max width and otherwise let it go up to 100% width (with some margins). also the left 50% and transform translate seems a bit of a roundabout way to center it on the page